src/components/LayoutTree.tsx

```diff
@@ -566,6 +568,10 @@ const LayoutTree = () => {
         ycol1Ancestors.push({ y: yaccum + height, depth: node.depth })
       }
 
+      if (node.isTableCol1) {
+        tableCol1WidthRef.current = Number(tableCol1Widths.get(head(parentOf(node.path)))) / fontSize || 0
```

Comment:
The `fontSize` division is unnecessary, as we can just specify it in px instead of em in the CSS `calc` below. Less work and less prone to rounding errors.